CloudFormation/network_Layer.yml

```
@@ -0,0 +1,136 @@
+      CidrBlock: 10.0.0.0/16
+      EnableDnsSupport: true
+      Tags:
+        - Key: Name
+          Value: lecture10
+
+# MapPublicIpOnLaunch は AWS マネジメントコンソールにおける「パブリックIPv4アドレスの自動割り当てを有効にする」にチェックを入れるかどうかに相当
+  PublicSubnet1a:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref lecture10
+      CidrBlock: 10.0.0.0/20
+      MapPublicIpOnLaunch: true
+      Tags:
+        - Key: Name
+          Value: PublicSubnet1a
+  
+  PrivateSubnet1a:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref lecture10
+      CidrBlock: 10.0.128.20/20
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet1a
+
+  PublicSubnet1c:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref lecture10
+      CidrBlock: 10.0.16.0/20
+      MapPublicIpOnLaunch: true
+      Tags:
+        - Key: Name
+          Value: PublicSubnet1c
+  
+  PrivateSubnet1c:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref lecture10
+      CidrBlock: 10.0.144.0/20
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet1c
+
+# InternetGatewayとRouteTableを作成
+  lecture10igw:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: lecture10-igw
+  AttachGateway:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId: !Ref lecture10
+      InternetGatewayId: !Ref lecture10igw
+
+  PublicRouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref lecture10
+      Tags:
+        - Key: Name
+          Value: PublicRouteTable
+  PrivateRouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref lecture10
+      Tags:
+        - Key: Name
+          Value: PrivateRouteTable
+
+# SubnetとRoutetableの関連付け
+  PublicRouteTableAssociation1a:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        RouteTableId: !Ref PublicRouteTable
+        SubnetId: !Ref PublicSubnet1a
+  PublicRouteTableAssociation1c:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        RouteTableId: !Ref PublicRouteTable
+        SubnetId: !Ref PublicSubnet1c
+  PrivateRouteTableAssociation1a:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        RouteTableId: !Ref PrivateRouteTable
+        SubnetId: !Ref PrivateSubnet1a
+  PrivateRouteTableAssociation1c:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        RouteTableId: !Ref PrivateRouteTable
+        SubnetId: !Ref PrivateSubnet1c
+
+# Routeの指定
+  PublicRoute:
+      Type: AWS::EC2::Route
+      Properties:
+        DestinationCidrBlock: 0.0.0.0/0
+        RouteTableId: !Ref PublicRouteTable
+        GatewayId: !Ref lecture10igw
+
+# Outputs
+Outputs:
+  lecture10:
+    Value: !Ref lecture10
+    Export:
+      Name: NetworkLayer-lecture10
```

Output名称は名前を取り決めた以外の人でも何が格納されているか名前からわかるべきだと思います。
この中には何が格納されますか？ID？Nameタグ？
もしNameならこんな感じが良いのではないでしょうか？
確認してみてください！
```suggestion
      Name: NetworkLayer-lecture10Name
```

---

出力内容を確認して中身が分かりやすい名称に変更しています。